practice-1/main.cpp

```diff
@@ -14,28 +15,19 @@ class Music {
     const string artist_name;
     unsigned int year;
 
-    Music() {
-        music_id = "";
-        artist_name = "";
-        year = 0;
-    };
+    Music() : music_id(""), artist_name(""), year(0) {};
```

Comment:
Writing it in this way removes the error because C++ does some magic behind the scenes.